onedal/utils/tests/test_validation.py

```diff
@@ -0,0 +1,117 @@
+# ==============================================================================
```

Comment:
A possibly dummy question:
Why this test needs to be added?
There is already https://github.com/intel/scikit-learn-intelex/blob/main/sklearnex/utils/tests/test_finite.py that tests about the same functionality.

Reply:
good question! that will be refactored in the next PR, this only creates the python oneDAL interface in the onedal folder and doesn't yet integrate it into sklearnex, so in short they test two different versions of ```_assert_all_finite```, the sklearnex tests are testing daal4py at the moment